.github/workflows/update-sbomscanner.yaml

```diff
@@ -0,0 +1,32 @@
+name: Sync SBOMscanner Helm chart
+
+on:
+  workflow_dispatch:
+  repository_dispatch:
+    types: [sbomscanner-release]
```

Comment:
```suggestion
    types: [release-sbomscanner]
```

Reply:
I wasn't using the same id as defined in https://github.com/kubewarden/sbomscanner/pull/524/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R131